polaris.shopify.com/pages/examples/content-block-width.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {ContentBlock, Box, Text} from '@shopify/polaris';
+
+import {withPolarisExample} from '../../src/components/PolarisExampleWrapper';
+
+function ContentBlockWidthExample() {
+  return (
+    <div style={{width: '500px'}}>
+      <ContentBlock width="medium">
```

Comment:
If we update `Width` type in `ContentBlock`, this needs to be updated
```suggestion
      <ContentBlock width="md">
```